bootstrap/kubeadm/main.go

```diff
@@ -216,18 +216,10 @@ func setupWebhooks(mgr ctrl.Manager) {
 		setupLog.Error(err, "unable to create webhook", "webhook", "KubeadmConfig")
 		os.Exit(1)
 	}
```

Comment:
We don't have to register those webhooks (we don't register any other List webhooks)